testing/gearmand/APKBUILD

```diff
@@ -0,0 +1,85 @@
+# Contributor: Aaron Hurt <ahurt@ena.com>
+# Maintainer: Aaron Hurt <ahurt@ena.com>
+pkgname=gearmand
+pkgver="1.1.12"
+pkgrel=2
+pkgdesc="Gearman job server and client"
+url="http://gearman.org"
+arch="all"
+license="BSD"
+depends="libgearman"
+depends_dev="boost-dev gperf libevent-dev util-linux-dev mariadb-dev
+	hiredis-dev openssl-dev sqlite-dev libmemcached-dev"
+makedepends="$depends_dev"
+install="$pkgname.pre-install $pkgname.post-deinstall"
+subpackages="$pkgname-doc libgearman libgearman-dev"
+source="https://launchpad.net/${pkgname}/1.2/${pkgver}/+download/${pkgname}-${pkgver}.tar.gz
+	libtest-cmdline.cc.patch
+	$pkgname.initd
+	$pkgname.confd"
+
+_builddir="$srcdir/$pkgname-$pkgver"
```

Comment:
It should be named `builddir`, this has been changed recently (you should upgrade your abuild package).